spec/dummy/db/schema.rb

```diff
@@ -31,7 +31,7 @@
 
   create_hypertable "events", "created_at", chunk_time_interval: "2 days"
 
-  add_hypertable_compression "events", "20 days", segment_by: "event_type, name", order_by: "occurred_at ASC, recorded_at DESC"
+  add_hypertable_compression_policy "events", "20 days", segment_by: "event_type, name", order_by: "occurred_at ASC, recorded_at DESC"
```

Comment:
We need to track the enable compression call first, I'm wondering if you get an error trying to setup your db without the enable call first 🤔